src/traces/histogram/calc.js

```diff
@@ -327,7 +328,7 @@ function calcAllAutoBins(gd, trace, pa, mainData, _overlayEdgeCase) {
             Lib.aggNums(Math.min, null, pos0);
 
         var dummyAx = {
-            type: pa.type === 'category' ? 'linear' : pa.type,
+            type: (pa.type === 'category' || pa.type === 'multicategory') ? 'linear' : pa.type,
```

Comment:
this is showing up a lot - I almost want to make something like `ax._isAnyCategory` (or perhaps `ax._isOrdinal`?) during `supplyDefaults`. Non-blocking, just something to consider.